lib/inspec/runner.rb

```diff
@@ -107,7 +107,8 @@ def render_output(run_data)
       return if @conf['reporter'].nil?
 
       @conf['reporter'].each do |reporter|
-        Inspec::Reporters.render(reporter, run_data)
+        result = Inspec::Reporters.render(reporter, run_data)
```

Comment:
This looks reasonable, but we've had so many issues around exit codes lately that I think we should be sure to include a functional test to check for regressions in the future and / or explicitly define expected behavior.

Reply:
Thank you for the attention on the functional tests @clintoncwolfe